Document java explicit span name with @WithSpan annotation (#6641)
open-telemetry/opentelemetry.io: content/en/docs/zero-code/java/agent/annotations.md

| name             | type              | default    | description                                                                                                                                  |
| ---------------- | ----------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------- |

> | `value`          | `String`          | `""`       | The span name, when empty (default) `<className>.<methodName>` will be used.                                                                 |

Reviewer: ```suggestion
| `value`          | `String`          | `""`       | The span name. If not specified, the default `<className>.<methodName>` is used.                                                                 |
```
